src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', errorMessage);
+};
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
```

Comment:
You are not rejecting the `promise1` promise. You should add a `reject` function to the `promise1` promise and call it if the `click` event doesn't occur within a certain time period.